/**:
  ros__parameters:
    # Frame ID
    base_frame_id: "base_link"

    # Velocity limits
    maximum_linear_velocity: 0.8  # m/s
    maximum_angular_velocity: 0.5  # rad/s

    # Vehicle parameters
    wheel_base: 0.65  # meters, example value

    publish_frequency: 30.0  # Hz